src/compiler/parser.ts

```diff
@@ -957,33 +960,33 @@ module ts {
         var parseErrorBeforeNextFinishedNode: boolean = false;
 
         sourceFile.syntacticDiagnostics = undefined;
-        sourceFile.referenceDiagnostics = [];
-        sourceFile.parseDiagnostics = [];
-        sourceFile.semanticDiagnostics = [];
+            sourceFile.referenceDiagnostics = [];

```

Comment:
VS diff/merge seems to be ignoring whitespace differences when it auto-merges and it results in odd indentation at times. Normally I'd just "Format Document" to clean up, but we have a number of places where we use idiosyncratic indentation (such as in https://github.com/Microsoft/TypeScript/pull/1781#discussion-diff-24618395 above),